### DISCUSSION TITLE
Would like to translate weblate into Toki Pona (tok)

### DISCUSSION THREAD

**Discussion by tbodt** [WeblateOrg/weblate#7253] [General]
What's the process for enabling a new language on https://hosted.weblate.org/projects/weblate?

**Comment by nijel** [comment]
See https://docs.weblate.org/en/latest/contributing/index.html#extending-lanugages.

I've added this one in https://github.com/WeblateOrg/language-data/commit/c8147a47d14565d3490c2b434e8ed29cec575a48

**Reply by tbodt** [reply]
Thanks. How to get it enabled in the Weblate project for translation of Weblate itself?

**Reply by nijel** [reply]
It needs new deploy to be visible on the server, it will take a couple of days. You should then be able to start the translation.

**Reply by tbodt** [reply]
I see. Thanks again

**Comment by comradekingu** [comment]
@tbodt https://hosted.weblate.org/projects/weblate/application/tok/
